Possibilità di scegliere la Provincia

albertograva opened the pull request:
Nella versione aggiornata dello script (catasto_unzip_merg_prov.py) è stata aggiunta possibilità di scegliere se procedere per singola regione o singola provincia (dando la possibilità di scelta). 
L'output è limitato al formato .gpkg.

albertograva committed:
Add files via upload

Script modificato per permettere la selezione di una solo provincia all'interno del file zip regionale, in modo da procedere con l'elaborazione (merge) solo di quella stessa provincia. Rimane comunque possibile procedere con il merge di tutte le province insieme.
Il formato di output ora è solo gpkg.
Alla fine delle elaborazioni tutti i file temporanei vengono rimossi.

Se si presenta l'errore "ERRORE: [SSL: DECRYPTION_FAILED_OR_BAD_RECORD_MAC] decryption failed or bad record mac (_ssl.c:2578)" è un problema momentaneo della rete durante il download.

albertograva committed:
readme v2

spiegazione modifiche

pigreco commented on README_v2.md:
@albertograva puoi spostare il file README v2 dentro la cartella dello script?

albertograva replied:
fatto!

albertograva committed:
Possibilità di scegliere la provincia

lo script catasto_unzip_merg_prov.py modifica lo script console_qgis.py, inserendo la possibilità di scegliere se procedere per singola regione o singola provincia (dando la possibilità di scelta). 
L'output è limitato al formato .gpkg.
Lo script cancella sempre i file temporanei creati nelle elaborazioni.

albertograva committed:
Delete README_v2.md

spostato
